### PR TITLE
WINDUPRULE-296: fixed java ee technology tags

### PR DESCRIPTION
**Pull request by mareknovotny**
fixing WINDUPRULE-296 after @OndraZizka catch it doesn't have what the doc specified.

### CODE REVIEW

**Comment by OndraZizka** [comment]
I'll leave up to Jess to confirm if this is how it should be. It matches how I understood it should be.

File: rules-reviewed/technology-usage/javaee-technology-usage.windup.xml [hunk 1]
```diff
@@ -58,41 +58,41 @@
...
         </rule>
-        <!-- JSON-P identication -->
+        <!-- JSON-P identification -->
         <rule id="javaee-technology-usage-00030">
             <when>
                 <javaclass references="javax.json.{*}"/>
             </when>
             <perform>
-                <technology-tag level="INFORMATIONAL">Java EE JSON-P</technology-tag>
+                <technology-tag level="INFORMATIONAL">JSON-P</technology-tag>
```
Comment (jsight) [inline rules-reviewed/technology-usage/javaee-technology-usage.windup.xml]: I think it is important to keep the "Java EE" prefix (or something like it). "JSON-P" is an overloaded acronym, and the more common usage isn't the one referenced here.

**Comment by jsight** [comment]
I just had one comment.... +1 to everything else.

**Comment by mareknovotny** [comment]
reverted back Java EE also for Batch just to be sure it is clear

**Pull request merged**